src/components/Modals/CreateItemModal/CreateItemModal.tsx

```diff
@@ -144,6 +144,7 @@ export default class CreateItemModal extends React.PureComponent<Props, State> {
         item = {
           id,
           name,
+          description: '',
```

Comment:
Do we need to add the category to this modal?

Reply:
The category is added in this modal already. It's inside `item.data.category`.

Reply:
I meant `description`. Solved off-line